modules/navier_stokes/test/tests/finite_volume/ins/turbulence/lid-driven/linear-segregated/lid-driven-turb-std-wall.i

```diff
@@ -0,0 +1,358 @@
+### Thermophysical Properties ###
```

Comment:
We only have a few options different in this, we should either do the `inlude` syntax and split the input file or just the cli_args.

Reply:
We have it the same way in the nonlinear. I see the interest in spelling options clearly and fully in one test. Keeping it as is for now

Reply:
I want complete test cases, because these are the cases from which people start from. I would prefer to keep this like it is. This is not a mistake, it is a conscious design decision based on feedback from users

Reply:
Users should use the Physics syntax. I don't see the need for complete inputs using the old syntax.

I dont want to be debugging 400 line input files because someone was inconsistent between their interpolation method, wall function, or wrote v = u instead of v = v